From dc1534b4b0cf2153e4b8848310efc8393fb73830 Mon Sep 17 00:00:00 2001
From: Benno Schulenberg <bensberg@telfort.nl>
Date: Sun, 12 Jun 2022 12:05:20 +0200
Subject: [PATCH] rules: remove two rules that reference two sections that were
 deleted

Commit fe36f31e8c from three months ago removed the need for special
aliases and key assignments for applealu_jis keyboards but forgot to
remove the two rules that reference these sections.

This fixes issue #325.

Signed-off-by: Benno Schulenberg <bensberg@telfort.nl>
---
 rules/0004-evdev.m_k.part | 1 -
 rules/0026-evdev.m_s.part | 1 -
 2 files changed, 2 deletions(-)

diff --git a/rules/0004-evdev.m_k.part b/rules/0004-evdev.m_k.part
index 1cff7b2f..62c21ffc 100644
--- a/rules/0004-evdev.m_k.part
+++ b/rules/0004-evdev.m_k.part
@@ -1,5 +1,4 @@
 ! model		=	keycodes
-  applealu_jis	=	evdev+macintosh(jisevdev)
  $jollamodels	=	evdev+jolla(jolla)
   olpc		=	evdev+olpc(olpc)
   olpcm		=	evdev+olpc(olpcm)
diff --git a/rules/0026-evdev.m_s.part b/rules/0026-evdev.m_s.part
index 2dd72051..80067025 100644
--- a/rules/0026-evdev.m_s.part
+++ b/rules/0026-evdev.m_s.part
@@ -2,5 +2,4 @@
   $evdevkbds    =   +inet(evdev)+inet(%m)
   chromebook    =   +inet(evdev)+inet(chromebook)
   ppkb          =   +inet(evdev)+inet(ppkb)
-  applealu_jis  =   +inet(evdev)+macintosh_vndr/jp(alujiskeys)
   *             =   +inet(evdev)
-- 
GitLab

